app/workers/facilities/pssg_download.rb

```diff
@@ -15,14 +13,17 @@ def perform
       begin
         download_data
       rescue => e
-        raise PSSGDownloadError, e.message
+        log_exception_to_sentry(e)
```

Comment:
Will this job ever retry after failure in Sidekiq if we rescue all the exceptions?

Reply:
I doubt it, unless sidekiq itself has some kind of error running the job

Reply:
I think avoiding retries is best for now. We just need to keep a close eye on sentry.